### PR TITLE
Functionality-openbare-stemming

### PR DESCRIPTION
**Pull request by snenenenenenene**
- added functionality to openbare stemming indicator
- added a conditional render statement to the vote overview due to municipalities such as Brugge sometimes not having a vote attached to the agenda item while the item is treated

### CODE REVIEW

**Review by aliokan — CHANGES_REQUESTED** [review]
Except the if statement around the voters table, everything looks good.

File: app/templates/agenda-items/agenda-item.hbs [hunk 1]
```diff
@@ -138,8 +138,18 @@
...
                 >
-                  <VoteOverview @vote={{this.model.vote}} />
-
+                  {{#if this.model.vote}}
+                    <VoteOverview
+                      @agendaItem={{this.model.agendaItem}}
+                      @vote={{this.model.vote}}
+                    />
+                  {{else}}
+                    <AuAlert
```
Comment (aliokan) [inline app/templates/agenda-items/agenda-item.hbs:147]: Can you reuse the logic removed in this PR: https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/103/commits/4cfa3873af05a6f5128139b108c6697d5be99e79#diff-c2141606cb920557e29cfaa8098d5b210e96f5756444fed2b47e7a216faf0244L144-L152

Comment (snenenenenenene) [reply app/templates/agenda-items/agenda-item.hbs:147]: If I use this logic Ember crashes when there is no vote found :/

Comment (aliokan) [reply app/templates/agenda-items/agenda-item.hbs:147]: Strange, the condition it based on `this.wasHandled`. I'm taking a look

Comment (aliokan) [reply app/templates/agenda-items/agenda-item.hbs:147]: I have not able to reproduce the bug. I commit the change 8acbf6732e264fe0cb8f3e6c824c81f7d51d5fb4

File: app/components/vote-overview.hbs [hunk 2]
```diff
@@ -1,8 +1,13 @@
 <p
   class="au-u-h6 au-u-flex au-u-flex--spaced-tiny au-u-flex--vertical-center au-u-margin-top"
 >
-  <AuIcon @icon="visible" @size="medium" />
-  Openbare stemming
+  {{#if @vote.secret}}
```
Comment (aliokan) [inline app/components/vote-overview.hbs:4]: @snenenenenenene I prefer to use ` @vote.secret` here

**Review by aliokan — APPROVED** [review]
@snenenenenenene I have made some modifications, it can be merged if it's good to you. :slightly_smiling_face:

**Pull request merged**